cli/fs_util.rs

```diff
@@ -3,6 +3,7 @@
 use deno_core::error::AnyError;
 use deno_core::error::Context;
 pub use deno_core::normalize_path;
+use deno_core::url::Url;
```

Comment:
I would prefer we use the `ModuleSpecifier` type alias when we are going to refer to something that we expect to be a module specifier.

Reply:
sgtm :)